beetsplug/fetchart.py

```diff
@@ -314,6 +314,16 @@ class CoverArtArchive(RemoteArtSource):
         GROUP_URL = 'http://coverartarchive.org/release-group/{mbid}/front'
 
     def get(self, album, plugin, paths):
+        """Cover Art Archive API offers pre-resized thumbnails at several sizes.
+        If the maxwidth config matches one of the already available sizes
+        fetch it directly intead of fetching the full sized image and resizing
+        it."""
```

Comment:
In Python, multi-line string literals are used as [docstrings](https://en.wikipedia.org/wiki/Docstring), not comments. So by putting this here, at the top of the method, you have replaced the documentation for the method—which I don't think you intended. This can just be a plain old comment (i.e., using `#`). And both the comment and the code should come after the method's docstring.